package.json

```diff
@@ -238,41 +239,22 @@
     "upload:macos": "oclif upload:macos",
     "upload:tarballs": "oclif upload:tarballs --xz",
     "upload:win": "oclif upload:win",
+    "readme": "oclif readme --no-aliases --repository-prefix '<%- repo %>/blob/<%- version %>/<%- commandPath %>'",
     "version": "oclif readme"
   },
   "types": "dist/index.d.ts",
   "devDependencies": {
     "@oclif/plugin-command-snapshot": "^4.0.2",
-    "@salesforce/dev-config": "^4.1.0",
-    "@salesforce/dev-scripts": "^5.10.0",
+    "@salesforce/dev-scripts": "^8.1.0",
     "@salesforce/plugin-release-management": "^4.5.4",
-    "@salesforce/prettier-config": "^0.0.3",
     "@salesforce/ts-sinon": "^1.4.19",
     "@types/debug": "^4.1.12",
-    "@typescript-eslint/eslint-plugin": "^5.62.0",
-    "@typescript-eslint/parser": "^5.62.0",
     "aws-sdk": "^2.1511.0",
-    "chai": "^4.3.10",
-    "eslint": "^8.55.0",
     "eslint-config-oclif": "^5",
     "eslint-config-oclif-typescript": "^2.0.1",
```

Comment:
is there really anything in here that we need?

Reply:
Hm, i don't know why we would need these. @mdonnalley do you know?